app/templates/components/recruiter-conversations/conversation.hbs

```diff
@@ -1,12 +1,25 @@
-{{#paper-card}}
-   {{#paper-card-title}}
-     {{#paper-card-title-text}}
-       <h2>You invited {{conversation.engineer.name}} to lunch!</h2>
-     {{/paper-card-title-text}}
-   {{/paper-card-title}}
-   <div class="md-actions" layout-align="center">
-     <a target="_blank" href="{{conversation.engineer.website}}"> {{#paper-button}}View their website{{/paper-button}}</a>
-     {{#link-to 'recruiter.view-conversation' conversation.id }}{{#paper-button}}View invitation{{/paper-button}}{{/link-to}}
-     {{#paper-button }}<div {{action 'deleteConvo'}}> Delete </div> {{/paper-button}}
+{{#paper-card as |card|}}
+  {{#card.title as |title|}}
+    {{#title.text as |text|}}
+      {{#text.headline}}
+        You invited {{conversation.engineer.name}} to lunch!
+      {{/text.headline}}
+    {{/title.text}}
+    {{#title.media size="lg" }}
+      {{recruiter-conversations/conversation/status conversation=conversation}}
+    {{/title.media}}
+  {{/card.title}}
+
+  <div class="md-actions" layout-align="center">
+    <a target="_blank" href="{{conversation.engineer.website~}}">
+      {{#paper-button}}View their website{{/paper-button}}
+    </a>
+    {{#link-to 'recruiter.view-conversation' conversation.id ~}}
+      {{#paper-button}}View invitation{{/paper-button}}
+    {{~/link-to}}
```

Comment:
Why is there both an `<a>` tag and a `{{link-to}}` seems like they do the same thing, lets use one syntax